style.css

```diff
@@ -22,45 +22,90 @@ body {
   background-color: #fff;
 }
 
-.header-nav {
+.desktop-container {
+  display: none;
+}
+
+.mobile-container {
+  width: 100%;
   display: flex;
   flex-direction: row;
-  flex-wrap: wrap;
+  align-items: center;
   justify-content: space-between;
-  width: 100%;
-  position: absolute;
-  height: 48px;
-  top: 15px;
+  background-color: white;
 }
 
 .logo {
-  padding: 10px 1px;
-  height: 15px;
-  margin-left: 12px;
-  align-items: center;
-  top: 5px;
-}
-
-.logo-text {
+  margin-left: 40px;
   color: #6070ff;
   font-family: "poppins", sans-serif;
   font-weight: 700;
   font-size: 18px;
-  text-align: center;
-  align-items: center;
-  margin-left: 10px;
 }
 
-.bars {
-  align-content: flex-end;
-  padding: 10px 1px;
-  height: 15px;
-  align-items: center;
-  margin-right: 12px;
+.hamburger.active span:nth-child(2) {
+  opacity: 0;
+  background-color: white;
 }
 
-.nav-items {
-  display: none;
+.hamburger.active span:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+  background-color: white;
```

Comment:
- [ ] It would be better if we follow the CSS best practices, we have a few repeating rules we need to address. A quick fix is to create a class for the same repeating rule and apply it on the HTML elements. For example; `background-color: white`